inference-engine/src/plugin_api/cpp_interfaces/interface/ie_internal_plugin_config.hpp

```diff
@@ -45,6 +45,12 @@ DECLARE_CONFIG_KEY(CPU_THREADS_PER_STREAM);
  */
 DECLARE_CONFIG_KEY(FORCE_DISABLE_CACHE);
 
+/**
+ * @brief This key determines if constant nodes should be dumped
+ * @ingroup ie_dev_api_plugin_api
+ */
+DECLARE_CONFIG_KEY(DUMP_CONSTANT_NODES);
```

Comment:
Why this cannot be enabled by default without an explicit option?

Reply:
Actually it is enabled by default in master right mow. The intention is to avoid dumping constant nodes both in performance counters and execution graph by default to make them more readable. However there still might be cases there this information might be useful, so the option is added.